src/index.js

```diff
@@ -90,6 +135,17 @@ const extendCodeBlockWithSyntaxHighlighting = ( settings ) => {
 						placeholder={ __( 'Write code…', 'syntax-highlighting-code-block' ) }
 						aria-label={ __( 'Code', 'syntax-highlighting-code-block' ) }
 					/>
+					<div aria-hidden={ true } className="code-block-bg">
+						{ attributes.content.split( '\n' ).map( ( v, i ) => {
```

Comment:
This seems to be causing an error for me when inserting a new Code block, as it appears `attributes.content` is `undefined` by default. So either that `default` value should be provided, or it it should be checked here.